File: 프로그래머스/0/120891. 369게임/369게임.js
function solution(order) {
    var answer = 0;
    
    const str = String(order);
    
    for(let s of str){
        if(s === "3")
            answer++;
        else if(s === "6")
            answer++;
        else if(s === "9")
            answer++;
    }
    
    return answer;
}

File: 프로그래머스/0/120891. 369게임/369게임.java
import java.util.Arrays;

class Solution {
    public int solution(int order) {
        return (int)Arrays.stream(String.valueOf(order).split(""))
            .filter(i -> {
                return i.equals("3") || i.equals("6") || i.equals("9");
            })
            .count();
    }
}